src/mcts/node.h

```diff
@@ -337,6 +337,14 @@ class EdgeAndNode {
   Node* node() const { return node_; }
 
   // Proxy functions for easier access to node/edge.
+  float GetQD(float default_q, float drawScore, bool logit_q = false) const {
+    float d = drawScore * GetD();
+    return (node_ && node_->GetN() > 0)
+               ?
+               // Scale Q slightly to avoid logit(1) = infinity.
+               (logit_q ? FastLogit(0.9999999f * (node_->GetQ() + d)) :
```

Comment:
This can result in FastLogit getting a value that is outside of -1,1 bounds.
Specifically if Q is already near -1 and drawScore * D is greater than 0

It is likely a very very rare case but something we might want to think of handling to be on the safe side.

Reply:
As Q approaches -1 or +1, D approaches 0. So I don't think that happens, but I am not sure.

Reply:
So long as drawScore is in the bounds -1 to 1, Q + draw_score*D should also be in the bounds [-1 to 1].

Reply:
W, D, L are all independent outputs I thought and had no requirement to sum up to 1

It seems that #956 adds in bounds for -1 to 1 so those might apply here as well.

Reply:
W D and L are not independent outputs - they are output from softmax - thus they sum to 1.